packages/react/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -88,7 +88,7 @@ const StyledLabel = styled.label`
 const ContentWrapper = styled.div<{ isExpanded: boolean, maxHeight?: number }>(({ isExpanded, maxHeight = 500 }) => `
     overflow: hidden;
     max-height: ${isExpanded ? `${maxHeight}px` : '0'};
-    transition: ${isExpanded ? 'max-height 1s ease-in' : 'max-height .5s ease-out'};
+    transition: ${isExpanded ? 'max-height 0.5s ease' : 'max-height 0.5s ease'};
```

Comment:
Enleve la condition, c'est la meme value. 

```suggestion
    transition: max-height 0.5s ease;
```

Reply:
C'est corrigé